hadoop-ozone/integration-test/pom.xml

```diff
@@ -103,6 +103,21 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
       <artifactId>junit</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-api</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-engine</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.junit.vintage</groupId>
+      <artifactId>junit-vintage-engine</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Should we add `junit-platform-launcher` also?

Reply:
The doc says `junit-platform-launcher` is required only for older IDEs, but I don't know what's "old".  I had no problem running tests in IDEA 2020.2 without it.

Reply:
2020.2 is new enough :-)
It would create problems for IDE bundled without junit-5 I guess, IntelliJ 2017 or before might get affected.
Do you any issues adding `junit-platform-launcher`? else having support for older IDE's won't hurt.